### PR TITLE
Lightbox can receive onLongPress function as a prop. 

### PR DESCRIPTION
**Pull request by TomJamesDuffy**

### CODE REVIEW

File: Lightbox.js [hunk 1]
```diff
@@ -117,6 +117,17 @@ export default class Lightbox extends Component {
     }
   }
 
+  onLongPress = (...args) => {
+    let result;
+    if (this.props.onLongPress) {
+      result = this.props.onLongPress(...args);
+    } else {
+      result = () => {}; // default behaviour is "do nothing"
```
Comment (peteroid) [inline Lightbox.js]: Can we just set it to null instead of create a new function in every render?

Comment (TomJamesDuffy) [reply Lightbox.js]: Good point, I think I was concerned about red screen of death if null was passed in but I just tested it and it seems to fail quietly.

**Comment by TomJamesDuffy** [comment]
Thanks for the pointer Eric, that is much simpler!

**Comment by ChrisEdson** [comment]
Hello! I think this would be a great feature to add in if possible. Should be a simple merge @oblador ?

**Comment by pzatorski** [comment]
Please merge this feature... @oblador

**Comment by sqlninja** [comment]
Really need this feature...
What is needed to get this merged?

**Pull request merged**